### PR TITLE
Update 508-accessibility-best-practices.md

### PR DESCRIPTION
**Pull request by jenstrickland**
Supplemented with additional details for testing and tooling; a future iteration will supplement with product strategy, user research, design, and qa best practices.

@1Copenut and @rianfowler — would you please review for accuracy?

### CODE REVIEW

**Review by rianfowler — COMMENTED** [review]
looks good to me but I have to defer to trevor for approval on some of the finer points about testing.

File: platform/accessibility/508-accessibility-best-practices.md
Comment (1Copenut) [inline platform/accessibility/508-accessibility-best-practices.md]: Let's remove this bullet point for now. The axe e2e script is way out of date and needs to be brought up to the new version of axe and Chrome Driver.

File: platform/accessibility/508-accessibility-best-practices.md
Comment (1Copenut) [inline platform/accessibility/508-accessibility-best-practices.md]: The table of contents link at the top of the page needs to be updated so this works properly. The text should match this heading, and the link link will look like this:

```html
[Manual Accessibility Testing](#manual-accessibility-testing)
```

**Review by 1Copenut — CHANGES_REQUESTED** [review]

**Comment by jenstrickland** [comment]
Changes accepted. Thank you!

**Pull request merged**
